Added Utterances integration

serafss2 opened the pull request:
Will enable users to comment using their github Auth0 integration.

Comments are created in a form of issues in github.

You can see a cool example here with the integration to this theme:
https://faresbessrour.com/misc/notcode/2020/02/16/Utterances-integration/

ashawley commented:
Would you add something to the README that documents this capability?  In particular, the possible values for the settings.  It may not be obvious from the Utterances web site what values are available for `issue-term`, `theme`.

ashawley commented on _config.yml:
```diff
@@ -40,6 +40,13 @@ fb_page_id:
+utterances_repo:
+utterances_issue-term: pathname
+utterances_theme: github-light
+utterances_crossorigin: anonymous
```
Remove `crossorigin`?  Doesn't seem it needs to be customized.

serafss2 replied:
Thanx for pointing that out! Removed it.

serafss2 commented:
I added some information in the `README.md`. I used the official description of Utterances to explain the feature. I also added the `themes` and the `issues-term` as you said, thanks for pointing that out @ashawley !
